packages/xo-web/src/common/custom-fields.js

```diff
@@ -28,17 +33,30 @@ const AddCustomFieldModal = decorate([
           [name]: value,
         })
       },
+      toggleType() {
```

Comment:
`toggleDate` maybe, because `toggleType` means that there is fields with type and fields without type.